app/views/play/SpectateView.coffee

```diff
@@ -180,9 +180,10 @@ module.exports = class SpectateLevelView extends RootView
     @insertSubView @tome = new TomeView levelID: @levelID, session: @session, otherSession: @otherSession, thangs: @world.thangs, supermodel: @supermodel, spectateView: true, spectateOpponentCodeLanguage: @otherSession?.get('submittedCodeLanguage'), level: @level, god: @god
     @insertSubView new PlaybackView session: @session, level: @level
 
-    @insertSubView new GoldView {}
+    goldInDuelStatsView = @level.get('slug') in ['wakka-maul', 'cross-bones']
```

Comment:
Can we reference ids instead of slugs here?

Reply:
This seems like data that should be attached to a level instead of a hard-coded list here.

Reply:
Those are good refactors to do once I'm sure how it's going to work; keeping it from interfering with production data and migrations for now and will set myself a task to refactor once we're happy with how it's working (along with some other cases like this). Until that point, using level slugs is less error-prone when tweaking which levels have what behavior while changing features.